docs/api-reference/algo-details-fastforest.md

```diff
@@ -28,4 +28,6 @@ For more see:
 * [Quantile regression
   forest](http://jmlr.org/papers/volume7/meinshausen06a/meinshausen06a.pdf)
 * [From Stumps to Trees to
-  Forests](https://blogs.technet.microsoft.com/machinelearning/2014/09/10/from-stumps-to-trees-to-forests/)
\ No newline at end of file
+  Forests](https://blogs.technet.microsoft.com/machinelearning/2014/09/10/from-stumps-to-trees-to-forests/)
+
+Check the See Also section for links to examples of the usage.
```

Comment:
"to examples of the usage" doesn't sound well.. 

I think they should all be "to usage examples."

Sorry..

Reply:
I agree with @sfilipi but we could live with what is here